src/common/scrub_types.h

```diff
@@ -207,4 +213,197 @@ struct scrub_ls_result_t {
 
 WRITE_CLASS_ENCODER(scrub_ls_result_t);
 
+template <>
```

Comment:
I'd suggest to add all formatters into the fmt snamespace. This prevents errors on some compiler versions.

Reply:
I think they are --

```
template <>
struct fmt::formatter<librados::object_id_t> {
```

Is your objection that you'd rather I use an enclosing namespace fmt{}?

Reply:
Hmmm... I don't think it was enough for the buggy g++, but I'll need to recheck.